src/messages.js

```diff
@@ -1,6 +1,7 @@
 export const MESSAGES = {
-    browserPathNotSet:  'Unable to run the browser. The browser path is not specified.',
-    unableToRunBrowser: 'Unable to run the browser. The file at {path} does not exist or is not executable.'
+    browserPathNotSet:       'Unable to run the browser. The browser path is not specified.',
+    unableToRunBrowser:      'Unable to run the browser. The file at {path} does not exist or is not executable.',
+    deviceSizeAliasNotFound: 'Unable to get device size. Device \'{deviceName}\' is not found.'
```

Comment:
Use double quotes around `{deviceName}`. Single quotes are not used for this purposes in the punctuation.